templates/username.html

```diff
@@ -5,10 +5,13 @@
 {% endblock %}
 
 {% block content %}
+
+{% include "publisher/_beta-notification.html" %}
+
 <section class="p-layout__main u-no-margin--top">
-  <div class="p-strip is-shallow">
+  <div class="p-strip is-shallow u-no-margin--top">
```

Comment:
Possibly not related directly to this PR, but layout on this page is strange.

There are no styles for `p-layout__main` at all, `u-no-margin--top` there is needed only to cancel Vanilla infamous `* + *`, but it seems that this whole `<section>` is not needed at all and p-strip with a row inside would be just fine (as it is in all other pages).

Also `u-no-margin--top` seems to not be needed (?)

Reply:
Fixed